pkgs/top-level/release.nix

```diff
@@ -13,6 +13,7 @@
   # The platforms for which we build Nixpkgs.
 , supportedSystems ? [ "x86_64-linux" "x86_64-darwin" "aarch64-linux" ]
 , limitedSupportedSystems ? [ "i686-linux" ]
+, limitedSupportedExtraConfigs ? [ "aarch64-unknown-linux-musl" "x86_64-unknown-linux-musl" ]
```

Comment:
Maybe name this relating to bootstrap tools since it's the only place this is used. Should lessen the confusion.

Reply:
If I renamed it, I'd want to rename `limitedSupportedSystems` as well, since it's also only used for bootstrapping.  But then that same parameter name is used in other places (with different values, no less), so should I rename them there too?  Is it worth it?

Reply:
Not sure I follow? I only see `systemsWithAnySupport` which is exclusively used for bootstrap-related things? Not terribly important though.

Reply:
I mean `limitedSupportedSystems` is used in other files.  (It's a parameter, after all.)